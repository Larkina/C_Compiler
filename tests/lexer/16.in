a a a a a a 
int *it = /*


comment

*/
4;