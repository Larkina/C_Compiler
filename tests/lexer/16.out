value=a, text=a, type=VAR
value=a, text=a, type=VAR
value=a, text=a, type=VAR
value=a, text=a, type=VAR
value=a, text=a, type=VAR
value=a, text=a, type=VAR
value=int, text=int, type=KEY_WORD
value=*, text=*, type=MUL
value=it, text=it, type=VAR
value==, text==, type=ASSIGN
value=4, text=4, type=INT
value=;, text=;, type=SEMICOLON